### PR TITLE
Agile Coretime Wiki Docs

### PR DESCRIPTION
**Pull request by DrW3RK**
Starter document for Agile Coretime and its terminology

### CODE REVIEW

File: polkadot-wiki/sidebars.js [hunk 1]
```diff
@@ -415,6 +415,20 @@ module.exports = {
                     "learn/learn-parachains-faq",
                   ],
                 },
+                {
+                  type: "category",
+                  label: "Agile Coretime",
+                  description: "Concepts, Implementation and Tutorials on Agile Coretime.",
+                  link: {
+                    type: 'generated-index',
+                    title: "Agile Coretime",
```
Comment (filippoweb3) [inline polkadot-wiki/sidebars.js:424]: are there more docs expected to come in this category? I can see the title of the category is the same as that of the doc

File: docs/learn/learn-agile-coretime.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+A system parachain that is responsible for the sale, manipulation of bulk coretime and 
+eventually the purchase of on-demand coretime credits. It is also responsible for 
+scheduling the respective cores on the relay chain to the parachains.
+
+### Timeslice
+
+A timeslice represents the granularity the Coretime Chain schedules cores on the Relay Chain. 
+The Coretime Chain announces a schedule for each core, comprising the next 80 relay 
+chain blocks in advance. The region length of 28 days corresponds to 5,040 timeslices.
```
Comment (filippoweb3) [inline docs/learn/learn-agile-coretime.md]: why is the 28 days important? Is this connected to bulk coretime?

File: docs/learn/learn-agile-coretime.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+### Task
+
+The term "Task" captures the abstraction of utility of the core. Tasks on the cores
+are not just limited to securing parachains.
+
+## Agile Coretime Implementation
+
+Coretime is managed through the [broker pallet](https://github.com/paritytech/polkadot-sdk/blob/eefd5fe4499515da66d088505093f75b5aa22550/substrate/frame/broker/src/lib.rs)
```
Comment (filippoweb3) [inline docs/learn/learn-agile-coretime.md]: Interesting, so the coretime chain will support other things than just coretime brokerage? Is that why they did not call it coretime broker chain?

File: docs/learn/learn-agile-coretime.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+This discussion on initial coretime pricing can be viewed [here](https://forum.polkadot.network/t/initial-coretime-pricing/5187)
+
+### Splitting and Interlacing
+
+Splitting and interlacing are actions that can be performed within a region. Performing either of 
+these actions makes you lose the right to a price-capped renewal.
+
+- **Splitting**: the action of dividing a region into several regions with different start and end 
+timeslices.
+- **Interlacing:** the action of dividing a region into the execution of different tasks for each
```
Comment (filippoweb3) [inline docs/learn/learn-agile-coretime.md]: Can we give a practical real example here about splitting and interlacing? It is a bit hard to follow

**Review by filippoweb3 — COMMENTED** [review]
Hey @DrW3RK I added some comment that could improve this page. It would be nice to start with an example and carry on that example through the sections.

File: polkadot-wiki/sidebars.js (hunk 1 above)
Comment (profradha) [reply polkadot-wiki/sidebars.js:424]: yes.

**Comment by DrW3RK** [comment]
Thanks for your suggestions. I will incorporate them in the docs and then open it up for review. Right now, it is not ready for a review.

File: docs/learn/learn-agile-coretime.md (hunk 2 above)
Comment (profradha) [reply docs/learn/learn-agile-coretime.md]: Yeah. Addressed it in the last commit, by rephrasing the sentence better.

File: docs/learn/learn-agile-coretime.md (hunk 3 above)
Comment (profradha) [reply docs/learn/learn-agile-coretime.md]: Of course. Other functionalities are planned, like adding support for coretime credits, etc., which could be part of a broker pallet or some other pallet. 

"why they did not call it coretime broker chain?"
This is partly answered in the "Product" section of FAQ uploaded. Also it is a lengthy name to have and some sort of branding decision for the system chain.

File: docs/learn/learn-agile-coretime.md (hunk 4 above)
Comment (profradha) [reply docs/learn/learn-agile-coretime.md]: Yeah, this will be part of another PR that provides more info through guides and tutorials.

File: docs/learn/learn-agile-coretime.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
+---
+id: learn-agile-coretime
+title: Introduction to Agile Coretime
+sidebar_label: Agile Coretime
+description: Introduction to Agile Coretime and its terminology
+keywords: [coretime, blockspace, parathread, parachain, cores]
+slug: ../learn-agile-coretime
+---
+
+Agile Coretime enables efficient utilization of Polkadot network resources and provides
```
Comment (filippoweb3) [inline docs/learn/learn-agile-coretime.md]: Should we implement conditional rendering for Kusama? I guess this page will be added to Kusama Guide as well?

File: docs/learn/learn-agile-coretime.md [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+keywords: [coretime, blockspace, parathread, parachain, cores]
+slug: ../learn-agile-coretime
+---
+
+Agile Coretime enables efficient utilization of Polkadot network resources and provides 
+economic flexibility for builders, generalizing Polkadot beyond what was initially proposed 
+and envisioned in its [whitepaper](https://polkadot.network/whitepaper/). 
+
+In Polkadot 1.0, the only way for a parachain to be secured by Polkadot was to rent a lease
+through an auction, which guaranteed parachain block validation for up-to two years. This involved
```
Comment (filippoweb3) [inline docs/learn/learn-agile-coretime.md]: we could link the auction page here. Although that page will be in the archive, but I think it would be informative to the reader

File: docs/learn/learn-agile-coretime.md [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+
+The figure below shows the core usage for Polkadot 1.0, where the horizontal axis is time, and each
+row represents a core. Colors show different parachains, each using one core (i.e., one parachain,
+one core).
+
+![core-usage-dumb](../assets/core-usage-dumb.png)
+
+Agile Coretime allows for the purchase of [coretime](#coretime) in “bulk” with an allocation for 
+one month. Heavy duty parachains which need to author a block every 12 
+seconds (or every 6s through [Async Backing](./learn-async-backing.md)), can seamlessly “renew” the [core](#core) each month.
```
Comment (filippoweb3) [inline docs/learn/learn-agile-coretime.md]:
```suggestion
seconds (or every 6s through [Asynchronous Backing](./learn-async-backing.md)), can seamlessly “renew” the [core](#core) each month. 
```

File: docs/learn/learn-agile-coretime.md [hunk 8]
```diff
@@ -0,0 +1,359 @@
...
+- Splitting and interlacing regions
+
+These features are still under development and will be rolled out in later releases: 
+
+- Getting rewards for adding your region to the on-demand pool
+- Credits for on-demand coretime (currently you are able to buy with native tokens of relay chain)
+
+#### What is currently being developed in the ecosystem to support Agile Coretime?
+
+There are several ecosystem projects in the making, such as the secondary markets for coretime,[Lastic](https://www.lastic.xyz/) (focusing on frontend) and [RegionX](https://app.regionx.tech/) (focusing on backend).
```
Comment (filippoweb3) [inline docs/learn/learn-agile-coretime.md]:
```suggestion
There are several ecosystem projects in the making, such as the secondary markets for coretime, [Lastic](https://www.lastic.xyz/) (focusing on frontend) and [RegionX](https://app.regionx.tech/) (focusing on backend). 
```

**Review by filippoweb3 — APPROVED** [review]
lgtm, I suggested conditional rendering for Kusama, and some minor edits

File: docs/learn/learn-agile-coretime.md (hunk 5 above)
Comment (profradha) [reply docs/learn/learn-agile-coretime.md]: Good Idea. But we shall do a separate PR for this.

File: docs/learn/learn-agile-coretime.md (hunk 6 above)
Comment (profradha) [reply docs/learn/learn-agile-coretime.md]: Added
